src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1417,5 +1417,26 @@ internal virtual IntermediateNode CreateIRCallNode(CallNode node, IRTranslatorCo
 
             return new IRCallNode(context.GetIRContext(node), this, args);
         }
+
+        /// <summary>
+        /// Function can override this method to provide BlankHandling policy for argument.
+        /// </summary>
+        /// <param name="index">0 based Index of argument.</param>
+        /// <returns></returns>
+        public virtual ArgPreprocessor GetArgPreprocessor(int index)
+        {
+            return ArgPreprocessor.None;
```

Comment:
>return ArgPreprocessor.None; [](http://example.com/codeflow?start=12&length=28)

Should this call the default?  If not, the code my want to document why

Reply:
updated the comments.